samples/redis/redis-feedback.include.md

```diff
@@ -0,0 +1,2 @@
+> [!NOTE]
+> Using Redis and NServiceBus together? [Tell us about it](https://github.com/Particular/NServiceBus/issues/6940).
```

Comment:
```suggestion
> Are you using Redis and NServiceBus together? [Tell us about it](https://github.com/Particular/NServiceBus/issues/6940); we'd love to hear more about your use cases.
```

Reply:
We can't use `we` or `we'd`. The engine prevents it.

Reply:
Maybe we need to relax those restrictions